web/src/main/bvmui/package.json

```diff
@@ -17,14 +18,24 @@
   "devDependencies": {
     "@mdi/font": "5.0.45",
     "@vue/cli-plugin-babel": "4.2.3",
+    "@vue/cli-plugin-eslint": "4.2.0",
     "@vue/cli-service": "4.2.3",
+    "@vue/eslint-config-standard": "5.0.1",
+    "babel-eslint": "10.0.3",
+    "eslint": "6.7.2",
+    "eslint-plugin-import": "2.20.1",
+    "eslint-plugin-vue": "6.1.2",
     "sass": "1.26.3",
     "sass-loader": "8.0.2",
     "vue-cli-plugin-vuetify": "2.0.5",
     "vue-template-compiler": "2.6.11",
     "vuetify-loader": "1.4.3",
     "webpack": "4.42.0"
   },
+  "resolutions": {
```

Comment:
This should fix https://github.com/diennea/bookkeeper-visual-manager/network/alerts